README.md

```diff
@@ -44,11 +44,33 @@ We heartily welcome external contributions to Marathon's documentation. Document
 
 ## Setting Up And Running Marathon
 
+### Dependencies
+Marathon has the following compile-time dependencies:
+* sbt - A build tool for scala. You can find the instructions for installing sbt for Mac OS X and Linux over [here](http://www.scala-sbt.org/0.13/tutorial/Setup.html).
+* JDK 1.8+
+
+For run-time, Marathon has the following dependencies:
+* libmesos - JNI bindings for talking to Apache mesos master. Look at the *Install Mesos* section for instructions to get libmesos.
+* Apache Zookeeper - We can have a spearate Zookeeper installation specifically for Marathon, or we can use the same Zookeeper used by Mesos.
```

Comment:
I'd use here the second-person singular (you), to be consistent with the rest of the documentation.